src/libraries/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoTests.cs

```diff
@@ -87,6 +87,20 @@ public void NativeDigitsTest()
             Assert.Equal(newDigits, nfi.NativeDigits);
         }
 
+        [Fact]
+        public void TestNFIFormatLimits()
+        {
+            foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.AllCultures))
+            {
+                NumberFormatInfo nfi = ci.NumberFormat;
+                Assert.True(nfi.CurrencyNegativePattern <= 16);
+                Assert.True(nfi.CurrencyPositivePattern <= 3);
+                Assert.True(nfi.PercentNegativePattern <= 11);
+                Assert.True(nfi.PercentPositivePattern <= 3);
+                Assert.True(nfi.NumberNegativePattern <= 4);
```

Comment:
These can all use Assert.InRange.  The benefit of that is, if it fails, the assert will output the invalid value that was tested, whereas with Assert.True it'll just output something like "Expected: True, Actual: False".